packages/salesforcedx-vscode-core/src/messages/i18n.ts

```diff
@@ -142,6 +142,10 @@ export const messages = {
     'No function.toml found. Create a function.toml, or create a new function using SFDX: Create Function.',
   force_function_start_warning_not_in_function_folder:
     'Open a function file to run SFDX: Start Function',
+  force_function_start_warning_plugin_not_installed:
+    'To run this command, install the evergreen plugin. For more info, see [Getting Started with Salesforce Functions](https://github.com/forcedotcom/evergreen-docs/blob/master/Getting%20Started%20Guide/Evergreen%20Getting%20Started%20Guide.md#installing-the-salesforce-functions-sfdx-plugin).',
```

Comment:
Ideally, we'd like to install the missing plugin automatically when user is prompted, but we don't have that pattern now. We could track that as an enhancement.

Reply:
Note "evergreen" should be replaced with "Salesforce Functions" (e.g. "To run this command, install the Salesforce Functions plugin.")

Reply:
Our doc style guide says to hyphenate "plug-in" but unsure if our messages consistently hyphenate it. So if you don't see it hyphenated elsewhere, let's not introduce an inconsistency

Reply:
@ekapner Yeah most of our mentions of "plug-in" are hyphenated. I'm going to leave as is